suites/pacific/upgrades/tier-2_scaleup_scaledown_after_upgrade_4x-to-5x-container.yaml

```diff
@@ -0,0 +1,598 @@
+#############################################################################
```

Comment:
Could we rename the file to `tier-2_dfg_test-elasticity-after-upgrade-from-4-to-latest.yaml`?

Reply:
Done.